### ISSUE TITLE
[CS - Accessibility Principles] Expand All - Collapse all seems missing

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-translations#171]
Expand All - Collapse all seems missing from https://www.w3.org/WAI/fundamentals/accessibility-principles/cs
